test/python/circuit/test_parameters.py

```diff
@@ -1254,6 +1255,22 @@ def test_bound_gate_to_matrix(self, gate_class):
 
         numpy.testing.assert_array_almost_equal(Operator(bound_circuit).data, gate.to_matrix())
 
+    def test_parameter_expression_grad(self):
```

Comment:
Can you add a test where `keys` is a list? 🙂

Reply:
sure